hudi-client/hudi-client-common/src/main/java/org/apache/hudi/keygen/KeyGenUtils.java

```diff
@@ -86,11 +86,11 @@ private static KeyGeneratorType inferKeyGeneratorTypeForAutoKeyGen(String partit
     if (!StringUtils.isNullOrEmpty(partitionFields)) {
       int numPartFields = partitionFields.split(",").length;
       if (numPartFields == 1) {
-        return KeyGeneratorType.SIMPLE;
+        return KeyGeneratorType.SIMPLE_KEYGEN;
       }
-      return KeyGeneratorType.COMPLEX;
+      return KeyGeneratorType.COMPLEX_KEYGEN;
```

Comment:
Not sure we have all these `_KEYGEN` as suffix because it is implied implicitly.

Reply:
Yeah earlier I had kept the same. But, while fixing some tests, I realized there is index type SIMPLE and keygen type SIMPLE and with static imports it might not be very readable. So, I added the suffix.

Reply:
> with static imports it might not be very readable.

Not very persuasive from my comprehension, just fix these two variables should be okay if you think they are confusing.